CHANGELOG.md

```diff
@@ -13,20 +13,24 @@ All changed fall under either one of these types: `added`, `changed`, `deprecate
 - splice-aware HISAT2 indexing for RNA-seq
 - quantifier HTSeq for RNA-seq
 - quantifier featurecounts for RNA-seq
+- Salmon will output a gene-level TPM matrix as well
 - added/expanded `seq2science explain` info (now covers RNA- and scATAC-seq too)
 - sequencing strandedness may now be inferred automatically (unless specified in the config/samples.tsv)
-- strandedness results are displayed in the multiQC under "Strandedness" 
+- strandedness results are displayed in the multiQC under "Strandedness"
+- a DEXSeq counts matrixs can now be generated with `dexseq: True`
+- seq2science CLI now has the same reason flag as snakemake (-r/--reason flag)
 
 ### Changed
 
-- rules and scriptnames in RNA-seq. ex: `txi.R` is now `quant_to_counts.R` to better reflect its function
+- rules and script names in RNA-seq. ex: `txi.R` is now `quant_to_counts.R` to better reflect its function
 - `quant_to_counts.R` now converts salmon transcript abundances to gene counts identically to DESeq2
 - STAR no longer outputs counts, and is no longer found under `quantifiers`
 - gene counts are generated from (filtered) bams when using either STAR or HISAT2 as aligner and HTSeq or featureCounts are quantifier
 - batch corrected gene counts are generated if a DESeq2 design contrast inclused a batch
 - batch corrected TPM are generated if a DESeq2 design contrast inclused a batch, and quantification was performed using Salmon
   - for us in ANANSE, for instance
 - `seq2science explain` now retrieves messages from `explain.smk`.
+- seq2science CLI now has default cores set to 2 (-j/--cores flag)
```

Comment:
I think this is a bad idea. I want people to consider the number of cores they want to use, and when setting it to 2, I am afraid lots of people will be waiting unnecessarily long. Why don't we set it to 2 (or any number) for a dry run, but for the real run people still have to set it?

Snakemake also switched that you HAVE to supply the nr of cores when running, so I guess they also had the same problem.